dll/win32/msgina/lang/en-US.rc

```diff
@@ -173,12 +173,19 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 146
+// Luna XP Style Dialog
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 135
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Luna-XP-style Shut Down dialog"
+CAPTION "Shut Down ReactOS"
```

Comment:
Shutdown is one word

Reply:
Look at 143 line: `CAPTION "Shut Down ReactOS"`

Reply:
`shutdown` is the noun, `shut down` is the verb, so this is fine.